### PR TITLE
[#1] [#3] - [Update README.md] - ["Initial commit" commit message does not pass conventional commits]

### LINKED ISSUE #1: Update README.md
Another special request that we should accommodate because this is when I'm updating the README.md from the GitHub UI.

https://github.com/ubiquity/devpool-directory/commit/538fe692775e6793aa6b01911bd256830656c924

### LINKED ISSUE #3: "Initial commit" commit message does not pass conventional commits
It is unfortunate that when creating a new repository from this template, the first commit will always fail CI. Is there a solution to change either the default commit message, or should we modify [our version of conventional commits](https://github.com/ubiquity/action-conventional-commits) to explicitly support the "Initial commit" commit message?

https://github.com/ubiquity/ubiquibot-sandbox/commit/6d5006d4594aaf4f79bab6f1767b29cce987a921

### PR DESCRIPTION
**Pull request by PhantomCracker**
Resolve #1 
Resolve #3

### CODE REVIEW

**Comment by PhantomCracker** [comment]
@pavlovcik please have a look when you have time

File: src/isValidCommitMesage.ts [hunk 1]
```diff
@@ -13,6 +13,7 @@ const DEFAULT_COMMIT_TYPES = [
     "merge",
     "update",
     "wip",
+    "Update: "
```
Comment (0x4007) [inline src/isValidCommitMesage.ts]: This doesn't seem to make sense to add. 

"Update README.md" doesn't contain the string "Update: "

```suggestion
    "Update "
```

File: .github/workflows/conventional-commits.yml
Comment (0x4007) [inline .github/workflows/conventional-commits.yml:1]: This file will need to be removed I guess.

**Review by 0x4007 — CHANGES_REQUESTED** [review]
There's no changes related to "Initial commit" so I don't see how that would be possible to have been solved. Please post proof. Lastly, you should open a new pull request per issue or else it makes reviews more complicated and confusing for me.

File: src/isValidCommitMesage.ts (hunk 1 above)
Comment (0x4007) [reply src/isValidCommitMesage.ts]: I see your proof though. How exactly does this work? https://github.com/ubiquity/action-conventional-commits/issues/1#issuecomment-1614455615

File: .github/workflows/conventional-commits.yml
Comment (PhantomCracker) [reply .github/workflows/conventional-commits.yml:1]: Removed

**Comment by PhantomCracker** [comment]
> There's no changes related to "Initial commit" so I don't see how that would be possible to have been solved. Please post proof. Lastly, you should open a new pull request per issue or else it makes reviews more complicated and confusing for me.

I agree, but was only a small change which was related to "Update" and this is why I did it like this; plus, we could had merge conflicts if I wouldn't do it in this way. From now on, I will create a new pull request for each issue, sorry.

Also, for "Initial commit" issue, it looks like I didn't push all the code :) sorry
Code updated

File: src/isValidCommitMesage.ts (hunk 1 above)
Comment (PhantomCracker) [reply src/isValidCommitMesage.ts]: Resolve the conversation based on the last comment from the pull request

**Comment by PhantomCracker** [comment]
@pavlovcik , here are all the proof of testings:

Initial commit: https://github.com/PhantomCracker/action-conventional-commits/actions/runs/5442577912/jobs/9897999333

Update file: https://github.com/PhantomCracker/action-conventional-commits/actions/runs/5442509145/jobs/9897833739

Test when removing "Update" from isValid constant: https://github.com/PhantomCracker/action-conventional-commits/actions/runs/5442701641/jobs/9898282183

File: src/isValidCommitMesage.ts [hunk 2]
```diff
@@ -13,12 +13,13 @@ const DEFAULT_COMMIT_TYPES = [
     "merge",
     "update",
     "wip",
+    "Update "
```
Comment (0x4007) [inline src/isValidCommitMesage.ts]:
```suggestion
```

File: src/isValidCommitMesage.ts [hunk 3]
```diff
@@ -13,12 +13,13 @@ const DEFAULT_COMMIT_TYPES = [
...
 ];
 
 const isValidCommitMessage = (message, availableTypes = DEFAULT_COMMIT_TYPES): boolean => {
     // Exceptions.
     // This is a message that's auto-generated by git. Can't do much about it unfortunately. Let's allow it.
-    const isValid = ["Merge ", "Revert ", "Update "].some((word) => message.startsWith(word));
+    const isValid = ["Merge ", "Revert ", "Update ", "Initial "].some((word) => message.startsWith(word));
```
Comment (0x4007) [inline src/isValidCommitMesage.ts]:
```suggestion
    const isValid = ["Merge ", "Revert ", "Update ", "Initial commit"].some((word) => message.startsWith(word));
```

**Review by 0x4007 — APPROVED** [review]

**Comment by 0x4007** [comment]
Looks like we already supported the commit message "Update README.md" but were using the wrong version of this CI. I'll provide credit for your efforts anyways. Thanks!

**Comment by PhantomCracker** [comment]
Thank you @pavlovcik! 👍

**Pull request merged**